packages/completer/src/widget.ts

```diff
@@ -498,6 +511,32 @@ export class Completer extends Widget {
     });
   }
 
+  private _updateDocpanel(): void {
+    let docpanel = this.node.querySelector('.jp-Completer-docpanel');
+    if (docpanel == null) {
```

Comment:
In JS, it's best practice to avoid `==` due to type coercion shenanigans.

If we want to return early if the value is "falsy", I'd recommend `if (!x)` instead.